analyses/survival-analysis/survival-analysis_immune.Rmd

```diff
@@ -99,19 +99,38 @@ quantiseq <- read_rds(
   select(-cell_type)
 
 # Read in histology file and filter to RNAseq samples with relevant columns
+# Will remove the below patients because I do not know if they had total or partial resections until we check with Jenn Mason
+# 7316-116 has Biopsy only;Gross/Near total resection
+# 7316-182 has Gross/Near total resection;Partial resection
+
 histology_df <- read_tsv(file.path(data_dir, "pbta-histologies.tsv"), guess_max = 10000) %>%
     filter(composition=="Solid Tissue",
            tumor_descriptor == "Initial CNS Tumor",
            experimental_strategy == "RNA-Seq") %>%
+    mutate(extent_of_tumor_resection = case_when(
+    extent_of_tumor_resection == "Biopsy only;Gross/Near total resection" ~ "Gross/Near total resection",
+    extent_of_tumor_resection == "Gross/Near total resection;Partial resection" ~ "Gross/Near total resection",
+    extent_of_tumor_resection == "Not Reported" ~ "Unavailable",
+    TRUE ~ as.character(extent_of_tumor_resection))) %>%
+  filter(sample_id != "7316-116" & sample_id != "7316-182") %>%
     select(Kids_First_Participant_ID, 
            Kids_First_Biospecimen_ID, 
            cancer_group, 
-           molecular_subtype, 
+           molecular_subtype,
+           extent_of_tumor_resection,
            OS_status, 
            OS_days, 
            PFS_days) %>%
     distinct() 
```

Comment:
I have similar comments here that I put in the analogous code chunk in #1342

Reply:
updated the recode for these pts